src/plugins/intel_gpu/src/kernel_selector/cl_kernels/strided_slice_ref.cl

```diff
@@ -289,33 +289,68 @@ KERNEL(strided_slice_ref)(OPTIONAL_SHAPE_INFO_ARG
 
 #if NEW_AXIS_MODE
     // If NEW_AXIS_MODE that just copy input to output
-#ifdef OUTPUT_LAYOUT_BFYX
+#ifdef INPUT0_LAYOUT_BFYX
+    const uint index_in_batch = (feature * get_global_size(2) + (uint)get_global_id(2))%(OUTPUT_SIZE_X * OUTPUT_SIZE_Y);
+    uint input_feature_id = (feature * get_global_size(2) +(uint)get_global_id(2)) / (OUTPUT_SIZE_X * OUTPUT_SIZE_Y);
     const uint w_input = 0;
     const uint z_input = 0;
-    const uint y_input = (uint)get_global_id(2) / INPUT0_SIZE_X;
-    const uint x_input = (uint)get_global_id(2) % INPUT0_SIZE_X;
-#elif OUTPUT_LAYOUT_BFZYX
+    const uint y_input = index_in_batch / OUTPUT_SIZE_X;
+    const uint x_input = index_in_batch % OUTPUT_SIZE_X;
+#elif INPUT0_LAYOUT_BFZYX
+    const uint index_in_batch = (feature * get_global_size(2) +(uint)get_global_id(2))%(OUTPUT_SIZE_X * OUTPUT_SIZE_Y * OUTPUT_SIZE_Z);
+    uint input_feature_id = (feature * get_global_size(2) +(uint)get_global_id(2)) / (OUTPUT_SIZE_X * OUTPUT_SIZE_Y * OUTPUT_SIZE_Z);
```

Comment:
nit: Fix code style according to the surrounding code
uint -> const uint
Spaces before and after operation signs